dist/common/scripts/scylla_prepare

```diff
@@ -99,6 +100,14 @@ def verify_cpu():
                     print('\nIf this is a virtual machine, please update its CPU feature configuration or upgrade to a newer hypervisor.')
                     sys.exit(1)
 
+def configure_aio_slots():
+    with open('/proc/sys/fs/aio-max-nr') as f:
+        aio_max_nr = int(f.read())
+    required_aio_slots = cpu_count() * 11026
+    if aio_max_nr < required_aio_slots:
+        with open('/proc/sys/fs/aio-max-nr', 'w') as f:
+            f.write(str(required_aio_slots))
```

Comment:
But this will leave nothing else for the rest of the system.
I'm not sure this will always work.
Maybe we need to add to it some slack (like the default value of 65536)